src/cards/repo-card.js

```diff
@@ -49,9 +49,11 @@ const getBadgeSVG = (label, textColor) => `
  *
  * @param {RepositoryData} repo Repository data.
  * @param {Partial<RepoCardOptions>} options Card options.
+ * @param {number} card_width The width of the card.
+ * @param {number} card_height The height of the card.
  * @returns {string} Repository card SVG object.
  */
-const renderRepoCard = (repo, options = {}) => {
+const renderRepoCard = (repo, options = {}, card_width, card_height) => {
```

Comment:
@airwakz I quickly reviewed this pull request, but it doesn't seem to work. I think the following things should be improved:

- The `card_width` and `card_height` variables should be contained in the `options` argument (see https://github.com/anuraghazra/github-readme-stats/blob/2567a6e580746a06b2aec999a7967ea08da7ec83/src/cards/top-languages-card.js#L723C4-L741 for an example).
- These options should also be accepted as query parameters in the [pin](https://github.com/anuraghazra/github-readme-stats/blob/master/api/pin.js) endpoint (see https://github.com/anuraghazra/github-readme-stats/blob/2567a6e580746a06b2aec999a7967ea08da7ec83/api/top-langs.js#L19 for an example).

Reply:
You can checkout the [CONTRIBUTING.md](https://github.com/anuraghazra/github-readme-stats/blob/fbcfb3e963209aecf978bb69b6181bf6a63941e3/CONTRIBUTING.md#local-development) guide on how you can debug your problems. If you have any questions, feel free to let me know 👍🏻.

Reply:
![image](https://github.com/anuraghazra/github-readme-stats/assets/114548361/f5701286-255a-4078-bc78-ee035cc23763)
I think this will work